app/views/browse/service-instance.html

```diff
@@ -145,6 +145,13 @@ <h3 ng-if-start="serviceClass.spec.description || serviceClass.spec.externalMeta
                           service-class="serviceClass"
                           service-plan="plan">
                         </service-instance-bindings>
+                        <div ng-if="(serviceInstance | serviceInstanceStatus) === 'Ready' && isMobileService">
```

Comment:
I have noticed there is an `isServiceInstanceReady` filter -> https://github.com/openshift/origin-web-common/blob/f53e010c74e3a20acabcd35b2b7676002c4ea96c/src/filters/resources.js#L133

Reply:
Thanks!